sale_order_lot_selection/wizards/stock_lot_add_to_sale_order.py

```
@@ -0,0 +1,92 @@
+            raise ValidationError(
+                _("Please select a customer to create a new Sale Order.")
+            )
+        self.sale_order_id = self.env["sale.order"].create(
+            {
+                "partner_id": self.partner_id.id,
+                "order_line": self._prepare_sale_order_line_vals(),
+            }
+        )
+        return self._action_open_sale_order()
+
+    def _prepare_sale_order_line_vals(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_prepare_sale_order_line_vals`

(ANN202)